src/directives/vLock.ts

```diff
@@ -0,0 +1,17 @@
+export const vLock = {
+    mounted(el: { classList: { add: (arg0: string) => void; remove: (arg0: string) => void; }; }, binding: { value: any; }) {
+        if (binding.value) {
+            el.classList.add('vlock')
+        } else {
+            el.classList.remove('vlock')
+        }
+      },
+      updated(el: { _magicSpinnerWrapper: { style: { display: string; }; }; classList: { add: (arg0: string) => void; remove: (arg0: string) => void; }; }, binding: { value: any; }) {
```

Comment:
I understand typing everything, but this is too much even for me 🤣

Reply:
Lol automatically done by VSC 😸 I will simplify it